xwiki-platform-core/xwiki-platform-like/xwiki-platform-like-ui/src/main/resources/XWiki/Like/UserProfileUIX.xml

```diff
@@ -254,30 +165,53 @@
     </property>
     <property>
       <content>{{velocity}}
-=$escapetool.xml($services.localization.render("like.userprofile.menu"))=
-#set($columns = ["doc.title", "doc.location", "like"])
+=$escapetool.xml($services.localization.render('like.userprofile.menu'))=
 #set ($docTitleDisplayName = $escapetool.xml($services.localization.render('like.livetable.column.title')))
 #set ($docLocationDisplayName = $escapetool.xml($services.localization.render('like.livetable.column.location')))
 #set ($likesDisplayName = $escapetool.xml($services.localization.render('like.livetable.column.likes')))
-#set($columnsProperties = {
-  "doc.title" : { "displayName": $docTitleDisplayName, "type": "text", "size": 30, "link": "view", "sortable": false, "filterable": false },
-  "doc.location" : { "displayName": $docLocationDisplayName, "type": "text", "size": 30, "html": true, "sortable": false, "filterable": false },
-  "like" : { "displayName": $likesDisplayName, "type" : "number", "sortable": false, "filterable": false }
-})
-
-#set ($currentUser = $services.model.serialize($doc.documentReference, "default"))
-#set ($pageToCallReference = $services.model.resolveDocument("XWiki.Like.UserProfileUIX"))
-#set ($pageToCall = $xwiki.getDocument($pageToCallReference))
-#set ($queryParams = {
-  "user": $currentUser,
-  "livetable": "true",
-  "xpage": "plain",
-  "outputSyntax": "plain"
-})
-#set($options = {
-  'url': $pageToCall.getURL('get', $escapetool.url($queryParams))
+#set ($currentUser = $services.model.serialize($doc.documentReference, 'default'))
+#set ($liveDataConfig = {
+  'meta': {
+    'propertyDescriptors': [
+       {
+         'id': 'doc.title',
+         'displayer': 'link',
+         'sortable': false,
+         'filterable': false,
```

Comment:
why don't we want that sortable or filterable?

Reply:
Because the result page does not support filtering and sorting.

Reply:
I guess that would be one improvement: to have a better way to write custom results page (in LD or LT or both) so that it's easy to support filtering and sorting, especially for well-known fields.

Reply:
>  to have a better way to write custom results page (in LD or LT or both) so that it's easy to support filtering and sorting, especially for well-known fields.

Don't forget that here the results are coming from a mix of two DBs results, so a generic solution probably wouldn't fit here.